src/freenas/etc/grub.d/10_truenas_linux

```diff
@@ -135,7 +136,7 @@ linux_entry ()
 	  init-*)
 	      title="$(gettext_printf "%s, with Linux %s (%s)" "${os}" "${version}" "${type#init-}")" ;;
 	  *)
-	      title="$(gettext_printf "%s, with Linux %s" "${os}" "${version}")" ;;
+	      title="$(gettext_printf "%s, with %s" "${os}" "${extitle}")" ;;
```

Comment:
I see there are other calls of `linux_entry` without `$5`, what will the title be in that case?

Reply:
The title would be Linux Kernel version string, which can be too long in some cases to be covered completely in that space provided in GRUB menu.

Reply:
I mean if `$5` is empty, then the title would be just `"TrueNAS SCALE 23.10, with "` which looks broken

Reply:
Thanks, I had not thought of that, I will update this.